libs/clients/vehicles/project.json

```diff
@@ -23,9 +23,9 @@
       "options": {
         "commands": [
           "curl -H \"X-Road-Client: IS-DEV/GOV/10000/island-is-client\" http://localhost:8081/r1/IS-DEV/GOV/10017/Samgongustofa-Protected/getOpenAPI?serviceCode=Mitt-Svaedi-V1 > src/clientConfig.json",
-          "cat <<< $(jq '.components.schemas.ProblemDetails.additionalProperties = false' src/clientConfig.json) > src/clientConfig.json",
-          "cat <<< $(jq '.paths.\"/VehicleReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json) > src/clientConfig.json",
-          "cat <<< $(jq '.paths.\"/OwnershipReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json) > src/clientConfig.json",
+          "jq '.components.schemas.ProblemDetails.additionalProperties = false' src/clientConfig.json > src/temp.json && mv src/temp.json src/clientConfig.json",
+          "jq '.paths.\"/VehicleReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json > src/temp.json && mv src/temp.json src/clientConfig.json",
+          "jq '.paths.\"/OwnershipReportPdf\".get.responses.\"200\".content.\"application/pdf\".schema.format = \"binary\"' src/clientConfig.json > src/temp.json && mv src/temp.json src/clientConfig.json",
```

Comment:
Is there any specific reason for these changes?

Reply:
It does exactly the same as the one before. But some reason I always have problems running the other one and after this change I have no problem running this along with others. I have made other try running this before without problems.